### PR TITLE
Adds tests so that GraphCypherQAChain class now has 100% coverage

### PR DESCRIPTION
**Pull request by alexthomas93**
# Description

Adds tests so that `GraphCypherQAChain` class now has 100% coverage

## Type of Change

- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Project configuration change

## Complexity

Medium

## How Has This Been Tested?

- [X] Unit tests
- [X] Integration tests
- [X] Manual tests

## Checklist

- [X] Unit tests updated
- [X] Integration tests updated
- [ ] CHANGELOG.md updated

### CODE REVIEW

File: libs/neo4j/tests/integration_tests/chains/test_graph_database.py [hunk 1]
```diff
@@ -4,10 +4,10 @@
 from unittest.mock import MagicMock
```
Comment (stellasia) [inline libs/neo4j/tests/integration_tests/chains/test_graph_database.py:4]: Detail: I think there is no `__init__.py` file in this directory, is this normal?

**Review by stellasia — APPROVED** [review]
Left a few comment but none is blocking to me, nice work!

File: libs/neo4j/langchain_neo4j/chains/graph_qa/cypher.py [hunk 2]
```diff
@@ -346,7 +346,7 @@ def from_llm(
         if validate_cypher:
             corrector_schema = [
                 Schema(el["start"], el["type"], el["end"])
-                for el in kwargs["graph"].structured_schema.get("relationships")
+                for el in kwargs["graph"].get_structured_schema.get("relationships", [])
```
[the remark this answers is not in the document]
Comment (alexthomas93) [reply libs/neo4j/langchain_neo4j/chains/graph_qa/cypher.py:349]: I agree it's not great. I think renaming it though would be a breaking change. Maybe we can change it in the next release

File: libs/neo4j/tests/integration_tests/chains/test_graph_database.py [hunk 3]
```diff
@@ -4,10 +4,10 @@
...
 
 from langchain_core.language_models import BaseLanguageModel
-from langchain_core.language_models.fake import FakeListLLM
 
 from langchain_neo4j.chains.graph_qa.cypher import GraphCypherQAChain
 from langchain_neo4j.graphs.neo4j_graph import Neo4jGraph
+from tests.unit_tests.llms.fake_llm import FakeLLM
```
[the remark this answers is not in the document]
Comment (alexthomas93) [reply libs/neo4j/tests/integration_tests/chains/test_graph_database.py:10]: > Not crucial, but maybe we should consider moving the `FakeLLM` class if used in both unit and integration tests.

Good point I'll move it somewhere more appropriate.

File: libs/neo4j/tests/integration_tests/chains/test_graph_database.py (hunk 1 above)
Comment (alexthomas93) [reply libs/neo4j/tests/integration_tests/chains/test_graph_database.py:4]: Have added one

File: libs/neo4j/tests/integration_tests/chains/test_graph_database.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (stellasia) [reply libs/neo4j/tests/integration_tests/chains/test_graph_database.py]: Thanks for the explanations! it all makes sense to me, no need to change anything.

**Pull request merged**